src/lib/types/validator.ts

```diff
@@ -15,3 +18,22 @@ export const zValidator = z
   }));
 
 export type Validator = z.infer<typeof zValidator>;
+
+export const zValidatorData = z
+  .object({
+    rank: z.number().nullable(),
+    rank_cummulative_voting_power: z.string().nullable(),
+    validator_address: zValidatorAddr,
+    account_address: zBechAddr20,
+    identity: z.string(),
+    moniker: z.string(),
+    details: z.string(),
+    commission_rate: z.coerce.number(),
+    is_jailed: z.boolean(),
+    is_active: z.boolean(),
+    voting_power: zBig,
+    uptime: z.number().optional(),
```

Comment:
From my undestanding, uptime can either be a number or zero. Therefore, let's eliminate the `optional` method.

Reply:
optional as it is used for validator info too